.dockerignore

```diff
@@ -9,4 +9,6 @@
 
 # unignored files
 !/satis.json
-!/LICENSE
\ No newline at end of file
+!/LICENSE
+!/mageos.html.twig
+!/package.json
```

Comment:
Why do we need this here? :thinking:

Reply:
I need the Twig file from the root of the project to be available to Satis when it is being built in the Docker entrypoint.
The package.json is in there so I can grab the current generator version from it, unless we use a different way of recording the version?

This bit of code here: https://github.com/mage-os/generate-mirror-repo-js/pull/87/files#diff-368e25a43c4a3200ecb9ae21e7b2a94da05d8eb24680bd6c2a37ee0e275bcd0aR89

Thanks